stable/prometheus-operator/templates/alertmanager/alertmanager.yaml

```diff
@@ -13,7 +13,7 @@ spec:
 {{- end }}
   replicas: {{ .Values.alertmanager.alertmanagerSpec.replicas }}
   listenLocal: {{ .Values.alertmanager.alertmanagerSpec.listenLocal }}
-{{- if .Values.global.rbac.create }}
+{{- if .Values.alertmanager.serviceAccount.name }}
```

Comment:
this is not right, the serviceAccount [will be created](https://github.com/helm/charts/blob/0c3053be740c5d9c3cb2446471ad92aac9bbf217/stable/prometheus-operator/templates/_helpers.tpl#L57-L64) even if `serviceAccount.name` is empty

Reply:
I see, it breaks the defaults, missed that

Reply:
Reworked the approach a bit, can't test it properly so I count on your reviews. Thanks!